### ISSUE TITLE
WeakReference in WatchingCache

### ISSUE THREAD

**Issue by darkleaf** [camsaul/methodical#28]
Is it possible remove WeakReference usage or add generic WatchingCache?
JavaScript engine does not implement WeakReference but  implement WeakMap.

For clojurescript support #20

**Comment by camsaul** [comment]
Yeah, we can make that Clojure-only with `#?(:clj)`. If it's not there it's not the end of the world in a web application.

**Comment by camsaul** [comment]
See #72. I think we should just drop the watching cache altogether
